src/back/controllers/api/routes/challenges/index.js

```diff
@@ -1,10 +1,9 @@
 'use strict';
 
 const router = require('express').Router;
-const getAll = require('./getAll');
-const post = require('./post');
-
+const controller = require('./controller');
```

Comment:
can you name not just controller, give name some context to handle logic clearfully